W-12610519-routing-changes-ch2 (#103)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ps-manage.adoc

You might want to add a route to enable a new subnet to be accessible or remove a route if you no longer want a subnet to be accessible through the transit gateway.

> You can remove the default routes that MuleSoft provides. However, when you change the default route, your private space might experience downtime. If the *X* icon doesn't appear for a route, you can't remove it. For private spaces created before March x, 2023, you cannot remove the default routes.

Author: note to self: insert date before merging.

> You can remove the default routes that MuleSoft provides. However, when you change the default route, your private space might experience downtime. If the *X* icon doesn't appear for a route, you can't remove it. For private spaces created before March x, 2023, you cannot remove the default routes.

Author: ```suggestion
When you create a new private space, you can remove the default routes that MuleSoft provides. However, when you change the default route, your private space might experience downtime. If the *X* icon doesn't appear for a route, you can't remove it. 

You cannot remove routes for existing private spaces (created before March 8, 2023). You must create and configure a new private space to remove the default routes that MuleSoft provides.
```

> You can remove the default routes that MuleSoft provides. However, when you change the default route, your private space might experience down time. If the *X* icon doesn't appear for a route, you can't remove it. Additionally, private spaces created before March x, 2023 cannot remove MuleSoft's default routes.

Reviewer: ```suggestion
You can remove the default routes that MuleSoft provides. However, when you change the default route, your private space might experience downtime. If the *X* icon doesn't appear for a route, you can't remove it. For private spaces created before March x, 2023, you cannot remove the default routes.
```

Reviewer: March x?

> MuleSoft also has routes, such as control plane traffic routes, that you cannot view or remove from the Route table.

Reviewer: ```suggestion
MuleSoft defines other routes, such as control plane traffic routes, that you cannot view or remove from the Route table.
```

> You cannot remove MuleSoft's defined routes for existing private spaces (created before March 8, 2023). You must create and configure a new private space to remove the default routes that MuleSoft defines. MuleSoft defines other routes, such as control plane traffic routes, that you cannot view or remove from the Route table.

Reviewer: Don't make MuleSoft (or any other trademark) possessive.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22MuleSoft%22

I don't see the need for the parenthetical. Can you just remove the parentheses?

Reviewer: Is there a reason "Route" is capitalized? If it's a UI element, also make it bold. If not, then lc.

Author: ```suggestion
You cannot remove the MuleSoft-defined routes for existing private spaces created before March 9, 2023. You must create and configure a new private space to remove the default routes that MuleSoft defines. MuleSoft defines other routes, such as control plane traffic routes, that you cannot view or remove from the route table.
```

Author: I don't really know of a good way to fix this without creating misplaced modifiers. Is this acceptable, @hannanelson ?

You might want to add a route to enable a new subnet to be accessible or remove a route if you no longer want a subnet to be accessible through the transit gateway.

When you create a new private space, you can remove the default routes that MuleSoft provides. However, when you change the default route, your private space might experience downtime. If the *X* icon doesn't appear for a route, you can't remove it. 

> You cannot remove routes for existing private spaces (created before March 8, 2023). You must create and configure a new private space to remove the default routes that MuleSoft provides.
> 
> MuleSoft defines other routes, such as control plane traffic routes, that you cannot view or remove from the Route table.

Author: ```suggestion
You cannot remove MuleSoft's defined routes for existing private spaces (created before March 8, 2023). You must create and configure a new private space to remove the default routes that MuleSoft defines. MuleSoft defines other routes, such as control plane traffic routes, that you cannot view or remove from the Route table.
```
